MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2117,10 +2121,9 @@ private string GetFixedSizeScalingFilter(Video3DFormat? threedFormat, int reques
             // If double rate deinterlacing is enabled and the input framerate is 30fps or below, otherwise the output framerate will be too high for many devices
```

Comment:
```suggestion
            // If double rate deinterlacing is enabled and the input framerate is 30fps or below, otherwise the output framerate will be too high for many devices.
```